src/Listeners/QueueableListener.php

```diff
@@ -17,12 +17,12 @@ public function shouldQueue(object $event): bool
     {
         if (static::$shouldQueue) {
             return true;
-        } else {
-            // The listener is not queued so we manually
-            // pass the event to the handle() method.
-            $this->handle($event);
-
-            return false;
         }
+
+        // The listener is not queued so we manually
+        // pass the event to the handle() method.
+        $this->handle($event);
+
+        return false;
```

Comment:
Same as the previous reviews, just reads better without the early return.

Reply:
[`fabc09a` (#1278)](https://github.com/archtechx/tenancy/pull/1278/commits/fabc09a16bf32fca9a42bb04df042a516de5d520)